vendor/ocaml-4.13-extended/parsetree.mli

```diff
@@ -90,6 +90,7 @@ and core_type =
      ptyp_loc: Location.t;
      ptyp_loc_stack: location_stack;
      ptyp_attributes: attributes; (* ... [@id1] [@id2] *)
+     ptyp_ext: string loc option;
```

Comment:
It is weird to have ext attached to core_type instead of core_type_desc. The same comment apply to other kind. Attaching to "desc" would make it easier to not forget any cases in fmt.ml